RUNNING_A_PROJECT.md

```diff
@@ -7,9 +7,16 @@ As a project manager you are expected to be available in the Slack channel for y
 
 We require the following:
 
+* We adhere to an *agile methodology* that includes these components:
+  - **There is a project story/tracking board** (see note below) that is maintained by the product owner (if there is one for the project) or the project manager/lead otherwise.  This provides a focal point for sprint planning and status tracking.
+  - **Scrums (aka standups) take place on a regular schedule** - every weekday, 3 times a week, etc. - whatever meets the needs of the particular project.  These are time-boxed (15 minutes) and follow the usual routine (What have you done since last scrum?, What will you be working on?, Do you have any blockers?).  It is normal for topics to be brought up during the scrum, which are placed in the "parking lot" to be discussed after the scrum concludes.  Also, the scrums serve as a good launching point for pair programming activity.
+  - **Development takes place in sprints**, the duration of which are particular to each project, but are typically one or two weeks in length.
```

Comment:
Also, another recommendation - for new projects, we should make it clear that we need an "Project Onboarding" meeting with the lead, shortly after the project is created.

There seems to be a meeting that mostly occurs on new projects like we did with wiki.edu - review the project, confirm non-profit and open sources, etc.  This meeting (or subsequent meeting) could also include a review of how AV project ideally could be run, and how *this* particular project can be managed to accommodate all/most/some of those practices.  

What we want to avoid is a run-away, unmanaged project, that presents a chaotic picture to AV members.